golos.publication/golos.publication.cpp

```diff
@@ -497,6 +504,12 @@ void publication::close_message(structures::mssgid message_id) {
         state.rsharesfn = new_rsharesfn.data();
     }
 
+    eosio_assert(mssg_itr->max_payout.symbol == state.funds.symbol, "LOGIC ERROR! publication::payrewards: mssg_itr->max_payout.symbol != state.funds.symbol");
+    payout = std::min(payout, mssg_itr->max_payout.amount);
```

Comment:
In case of last message with 0 payout we will lost tokens in the fund without messages.
I request to check `std::min(payout, mssg_itr->max_payout.amount);` only in `else {`